ui/pages/confirmations/hooks/useCurrentConfirmation.ts

```diff
@@ -46,9 +55,11 @@ const useCurrentConfirmation = () => {
     selectUnapprovedMessage(state, confirmationId),
   );
 
-  const isCorrectTransactionType = REDESIGN_TRANSACTION_TYPES.includes(
-    transactionMetadata?.type as TransactionType,
-  );
+  const isCorrectTransactionType =
```

Comment:
Does the user setting not matter here?

Reply:
I moved this discussion to slack, follow it [here](https://consensys.slack.com/archives/C070G0RCX0T/p1720690681341889)

Reply:
Yes! The toggle will be done separately and this looks good as is.

Reply:
To clarify, we insist on the user setting below regardless of whether the approval type or transaction type matches.